spec/methods/auth/auth.ts

```diff
@@ -34,6 +37,27 @@ export type UserOrChallengeResult = XOR<AuthData, Challenges.Data>;
  */
 export type UserOrChallengeResponse = ApiResponse<UserOrChallengeResult>;
 
+/**
+ * Response from a call to begin
+ * App MFA setup.  It contains a secret code
+ * to enter into the MFA App.
+ */
+export type SecretCodeResult = {
```

Comment:
Now that I look over this again, it doesn't seem like this `Result` is reused anywhere.  Wanna just define it inline?